src/extend.js

```diff
@@ -29,5 +33,13 @@ module.exports = function(protoProps, staticProps) {
   // call it with the extended child object.
   if (isFunction(Parent.extended)) Parent.extended(Child);
 
+  // Create a parsedIdAttribute for use when setting `model.id` in `Model.set`
+  if (isFunction(Child.prototype.parse)) {
+    tmpParsedObj = Child.prototype.parse({[Child.prototype.idAttribute]: null});
```

Comment:
It is possible to set `parsedIdAttribute` on a single line, but I broke it up here for increased readability.